guides/_subsections/snapshot-a-recording.md

```diff
@@ -1,15 +1,17 @@
 ## [Snapshot a Recording](#snapshot-a-recording)
-Snapshotting an active recording produces a new recording in the target JVM
-named `snapshot-n`, where `n` is a natural number. This snapshot contains all of
-the JFR data that was present in the target JVM at the time that the snapshot
-was taken and is in the `STOPPED` state as soon as it is created (that is, the
-snapshot will never record any further information and is fixed in size).
-This is useful when you have an ongoing, continuous recording active in a
-target application and want to preserve the specific information at a given
-point in time. If the continuous source recording is configured with a maximum
-event age or maximum recording size then the need for snapshots is more
-apparent, since without snapshotting some older data will eventually be dropped
-from the source recording and lost.
+Snapshotting an application produces a new Flight Recording named `snapshot-n`,
+where `n` is a natural number. This snapshot contains all of the JFR data that
+was present in the target JVM at the time that the snapshot was taken and is in
+the `STOPPED` state as soon as it is created (that is, the snapshot will never
+record any further information and is fixed in size). This is useful when you
+have an ongoing, continuous recording active in a target application and want
+to preserve the specific information at a given point in time. If the
+continuous source recording is configured with a maximum event age or maximum
+recording size then the need for snapshots is more apparent, since without
+snapshotting some older data will eventually be dropped from the source
+recording and lost. It is also useful when you have multiple concurrent source
+recordings running and want an easy way to capture the sum of all of their data
```

Comment:
since without snapshotting, some older data will eventually be dropped and lost from the source recording.